README.adoc

```diff
@@ -199,4 +199,15 @@ image::media/bill-similarity-compare.png[BilltoBillSimilarity,300,200]
 .Text-to-bill Similarity 
 [#img-text-to-bill-similarity]
 [caption="Figure 4: " ]
-image::media/bill-similarity-text-search.png[TextToBillSimilarity,300,200]
\ No newline at end of file
+image::media/bill-similarity-text-search.png[TextToBillSimilarity,300,200]
+
+
+## Relevant Committee Documents
+
+To load Relevant Committee Documents data use the following instructions:
+
```

Comment:
Please add documentation in SCRAPING.adoc for what data is scraped, how to initiate the data. Also in UPDATES_CELERY.adoc to describe the task you've set up for SAP.